apps/ff-site/README.md

```diff
@@ -157,11 +110,7 @@ This setup is only necessary for production. For local development, follow the i
 
 #### Working with the CMS locally
 
-You can connect Decap CMS to the local Git repository. To do this, follow these steps:
-
-1. Run `npx decap-server`
-2. Run `npm run dev`
-3. Open http://localhost:3000/admin/index.html
+You can connect Decap CMS to the local Git repository using Decap's local server by running `npx decap-server`. This will bypass the github authentication and allow you to work with the CMS locally.
```

Comment:
```suggestion
You can connect Decap CMS to the local Git repository using Decap's local server by running `npx decap-server`. This will bypass the GitHub authentication and allow you to work with the CMS locally.
```